static/css/components/list-entry.less

```diff
@@ -3,9 +3,11 @@
  * https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#listentry
  */
 .listEntry {
-  width: 280px;
-  margin: 0 30px 35px 0;
-  float: left;
+  width: 100%;
+  @media only screen and (min-width: @width-breakpoint-tablet) {
+    width: 280px;
+  }
+  margin: 20px 10px;
```

Comment:
Always put a space after selectors (not sure why stylelint doesn't complain about that)